### PR TITLE
Product Feed: Add configurable image size and additional photos

### PR DESCRIPTION
**Pull request by krzysztofwolski**
## Scope of the PR

- add users control over the size of images used in the feed. The default image size increased to 1024
- added additional images from the product gallery to the feed

The PR can be reviewed, but please do not merge - I'm waiting for `robots.txt` change used by Saleor instances. Without it Google crawlers are unable to fetch images.

## Checklist

- [x] `.github/dependabot.yaml` is up-to date.
- [x] I added changesets and [read good practices](/.changeset/README.md).

### CODE REVIEW

File: apps/products-feed/src/modules/app-configuration/app-config.ts [hunk 1]
```diff
@@ -1,6 +1,14 @@
 import { createLogger } from "@saleor/apps-shared";
 import { z } from "zod";
 
+const imageSizeFieldSchema = z.coerce.number().gte(256).default(1024);
```
Comment (krzysztofwolski) [inline apps/products-feed/src/modules/app-configuration/app-config.ts:4]: FYI: `coerce` is needed to fix issues with client-side validation. Without it input fields are rejected as values are submitted as strings

File: apps/products-feed/src/modules/app-configuration/image-configuration-form.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    resolver: zodResolver(imageSizeInputSchema),
+  });
+
+  const options = [
+    { value: "256", label: "256px" },
+    { value: "512", label: "512px" },
+    { value: "1024", label: "1024px" },
+    { value: "2048", label: "2048px" },
+    { value: "4096", label: "4096px" },
+  ];
```
Comment (lkostrowski) [inline apps/products-feed/src/modules/app-configuration/image-configuration-form.tsx:29]: move options outside of the component so we keep stable reference

File: apps/products-feed/src/modules/app-configuration/image-configuration-form.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+  return (
+    <Box
+      as={"form"}
+      display={"flex"}
+      gap={5}
+      flexDirection={"column"}
+      onSubmit={handleSubmit((data) => {
+        props.onSubmit(data);
```
Comment (lkostrowski) [inline apps/products-feed/src/modules/app-configuration/image-configuration-form.tsx:38]: non blocking: lambda not required

**Review by lkostrowski — APPROVED** [review]
comments non blocking

**Review by lkostrowski — APPROVED** [review]

**Pull request merged**